shell.nix

```diff
@@ -55,6 +55,7 @@ mkShell {
     python3
     utillinux
     xfsprogs
+    etcd
```

Comment:
would be good to keep alphabetical order

Reply:
good point, I sorted the whole list now